mmv1/third_party/validator/tests/data/example_vpc_access_connector.tfplan.json

```diff
@@ -49,7 +49,8 @@
                     "id": true,
                     "project": true,
                     "self_link": true,
-                    "state": true
+                    "state": true,
+                    "connected_projects": true
```

Comment:
Thank you for help fixing the missing field!
For my own education purpose: Is the ordering of the "connected_projects" returned by the GCP API guaranteed? If not, do we need some sort of custom diff so that the "Connector" resource is considered unchanged when the ordering of the "connected_projects" has change?

Reply:
FYI, I was looking at this line as reference: https://github.com/GoogleCloudPlatform/magic-modules/blob/241f454d6633a81984a278ffc85b6ad2909d9ff3/mmv1/products/compute/Subnetwork.yaml#L190

Reply:
+1 to this! The PR looks good but we'd want to identify any list ordering issues before merging

Reply:
Good point. Added `unordered_list: true` to the new field.